docs/understanding/pipeline.rst

```diff
@@ -0,0 +1,30 @@
+.. include:: ../links.inc
+
+.. _pipeline:
+
+The Junifer Pipeline
+====================
+
+The junifer pipeline is the main execution path of junifer. It consists of five steps:
+
+1. :ref:`Data Grabber <datagrabber>`: Interpret the database and provide a list of files.
```

Comment:
`DataGrabber`

Reply:
I stil prefer to use the two words to describe the concept and not the Class name.

Reply:
In the Understanding section, we have `DataGrabber` for the concept as well. Again my reasoning is that if we keep it as one name throughout, users are not confused. And also, one can mentally link better to the name of the step being the class category's name.